### PR TITLE
ISSUE 2 - fix integration tests

### PR DESCRIPTION
**Pull request by javamonkey79**

### CODE REVIEW

**Comment by samshuster** [comment]
nice job just confirmed it works

File: src/main/java/com/edmunds/tools/databricks/maven/BaseDatabricksMojo.java [hunk 1]
```diff
@@ -21,63 +21,89 @@
...
+                        .build();
             } else {
                 throw new IllegalArgumentException("Must either specify user/password or token!");
             }
         }
         return databricksServiceFactory;
     }
 
+    private void loadPropertiesFromSystemEnvironment() {
+        String envUser = System.getenv(DB_USER);
```
Comment (samshuster) [inline src/main/java/com/edmunds/tools/databricks/maven/BaseDatabricksMojo.java]: so i am ok with this as it is. A gotya for users is that user password will always trump token access, How about add a comment to the token property above?

Comment (javamonkey79) [reply src/main/java/com/edmunds/tools/databricks/maven/BaseDatabricksMojo.java]: That works for me.

Comment (javamonkey79) [reply src/main/java/com/edmunds/tools/databricks/maven/BaseDatabricksMojo.java]: Done. Check again.

File: src/it/simple-it/src/main/resources/databricks-plugin/databricks-job-settings.json [hunk 2]
```diff
@@ -22,14 +22,17 @@
       "enable_elastic_disk": false
     },
     "email_notifications": {
+      "on_failure": [
+        "fake@fake.com"
```
Comment (l0s) [inline src/it/simple-it/src/main/resources/databricks-plugin/databricks-job-settings.json]: Very minor: AFAIK, this is a valid email address. Consider using something that will never resolve, like "fake@example.com".

Comment (javamonkey79) [reply src/it/simple-it/src/main/resources/databricks-plugin/databricks-job-settings.json]: Oh interesting. I'll change it next time.

**Pull request merged**